exo/inference/inference_engine.py

```diff
@@ -17,6 +17,7 @@ async def infer_tensor(self, request_id: str, shard: Shard, input_data: np.ndarr
 
 
 def get_inference_engine(inference_engine_name: str, shard_downloader: 'ShardDownloader'):
+  print(f"get_inference_engine called with: {inference_engine_name}")  # Debug print
```

Comment:
if you want a log here, please add `if DEBUG >= 2:`

Reply:
Done Thanks.